.github/pull_request_template.md

```diff
@@ -1,32 +1,17 @@
-Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], or [CONTRIB]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).
 
-Changes proposed in this pull request:
--
--
--
 
 
-After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.
+- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
+- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
+- [ ] Code is linted
 
-For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.
+    ```
+    black .
 
-In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in GitHub issues or Slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).
+    ruff . --fix
```

Comment:
How about `invoke lint`?

Reply:
```
- [ ] Code is linted (run `invoke lint`)
```
or something to that effect

Reply:
Just being consistent with https://docs.greatexpectations.io/docs/contributing/style_guides/code_style/#linting and other places like https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations/#8-linting

Reply:
Want fewer things confusing users. Sometimes invoke tasks lead to me needing more troubleshooting, but the lint task usually isn't one of them.

Reply:
What about `invoke fix`?